modules/apps/transfer/keeper/msg_server_test.go

```diff
@@ -61,9 +61,11 @@ func (suite *KeeperTestSuite) TestMsgTransfer() {
 		res, err := suite.chainA.GetSimApp().TransferKeeper.Transfer(sdk.WrapSDKContext(suite.chainA.GetContext()), msg)
 
 		if tc.expPass {
+			suite.Require().NotEqual(res.Sequence, uint64(0))
 			suite.Require().NoError(err)
 			suite.Require().NotNil(res)
 		} else {
+			suite.Require().Equal(res.Sequence, uint64(0))
```

Comment:
Looks like there's a `nil` pointer happening here in tests. We can't make the assertion at this level as `if err != nil` we return `nil` for the `res`.

Reply:
Whoops, thanks. Fixed.